packages/opentelemetry-metrics/src/export/Controller.ts

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { unrefTimer } from '@opentelemetry/core';
+import { Meter } from '../Meter';
+import { MetricExporter } from './types';
+import { ExportResult } from '@opentelemetry/base';
+
+const DEFAULT_EXPORT_INTERVAL = 60_000;
+
+export class Controller {}
```

Comment:
Why are we extending an empty class?

Reply:
Currently, only `PushController` is included which triggers collection using a fixed period. In the future, we might include the`PullController`. This `Controller`  class created as a placeholder for future work. Maybe I will remove it now and include it later as per the need. WDYT?

Reply:
Think I'm torn on this. It's probably fine to keep it for now.